simple-jndi/jdbc.properties

```diff
@@ -0,0 +1,10 @@
+DbOmegaEtl/type=javax.sql.DataSource
```

Comment:
Is there any way to set the data dir to a temporary path. i.e. could this be done programatically at runtime rather than coding it into a properties file?